packages/cli/src/index.ts

```diff
@@ -78,8 +80,16 @@ cli.command('serve', 'serve the app locally')
     .option('--allowExternal, --allow-external', '[boolean] allow external IPs to access the server', {
         default: false,
     })
+    .option('--appType [appType], --app-type', '[string] specify app type. Overrides manifest values')
     .action(async (options) => {
-        await createDevelopmentServer(options.entryPath, options.port, options.allowExternal);
+        const manifest = reactiveJson<AppManifest>(join(process.cwd(), 'manifest.json'));
+        const appType = options.appType ?? manifest.appType;
```

Comment:
```suggestion
        const appType = options.appType || manifest.appType;
```
`'' | 'theme' === ''`